web/crux/src/app/audit/audit.service.ts

```diff
@@ -26,7 +26,7 @@ export default class AuditService {
     const identites = await this.kratos.getIdentitiesByIds(auditLog.map(it => it.userId))
 
     return {
-      data: auditLog.map(it => this.mapper.toGrpc(it, identites)),
```

Comment:
We are using both of the formats, talk about this in person and refactor.

Reply:
i didn't realized that one, while renaming it. :smile: It's indeed sounds silly. However there is already a `configToDagentConfig`. I rename them.